src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2723,7 +2723,64 @@ export class ResultsViewPageStore extends AnalysisStore
         default: [],
     });
 
-    readonly clinicalAttributes_customCharts = remoteData({
+    readonly plotClinicalAttributes = remoteData<ExtendedClinicalAttribute[]>({
+        await: () => [
+            this.studyIds,
+            this.clinicalAttributes_profiledIn,
+            this.clinicalAttributes_comparisonGroupMembership,
+            this.samples,
+            this.patients,
+        ],
+        invoke: async () => {
+            const serverAttributes = await client.fetchClinicalAttributesUsingPOST(
+                {
+                    studyIds: this.studyIds.result!,
+                }
+            );
+            const specialAttributes = [
```

Comment:
@gblaih @inodb  do you understand this here?  why are we adding these hardcoded things here?

Reply:
Yes, it is correct.

Reply:
Thanks @gblaih and @Nelliney ! Just talked to @gblaih and he will take a look at changing this a bit to avoid code duplication